chia/wallet/wallet_node.py

```diff
@@ -225,6 +227,60 @@ async def get_private_key(self, fingerprint: Optional[int]) -> Optional[PrivateK
 
         return key
 
+    def set_resync_flag(self, fingerprint, flag: bool = True):
+        config = load_config(self.root_path, "config.yaml")
```

Comment:
Should probably use the context manager version:
`with lock_and_load_config(self.root_path, "config.yaml") as config:`